GitUI/GitUI.csproj

```diff
@@ -41,62 +41,30 @@
     <DocumentationFile>bin\Release\GitUI.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="JetBrains.Annotations, Version=11.1.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325, processorArchitecture=MSIL">
-      <HintPath>..\packages\JetBrains.Annotations.11.1.0\lib\net20\JetBrains.Annotations.dll</HintPath>
+    <Reference Include="ICSharpCode.TextEditor">
+      <HintPath>..\Bin\ICSharpCode.TextEditor.dll</HintPath>
```

Comment:
This looks like it has been built from this project https://github.com/KindDragon/ICSharpCode.TextEditor

@KindDragon is looks like there was only one functional change made to the code https://github.com/KindDragon/ICSharpCode.TextEditor/pull/3. 
Is there anything that precludes us from referencing the official nuget package https://www.nuget.org/packages/ICSharpCode.TextEditor/?

Reply:
➡️ This would be a separate change (but otherwise in favor of it if possible)

Reply:
> Is there anything that precludes us from referencing the official nuget package https://www.nuget.org/packages/ICSharpCode.TextEditor/?

We used official package before, but we need some patches to make it work for us https://github.com/KindDragon/ICSharpCode.TextEditor/commit/ae3a7fdfc03d5dc3fe424896ef5a60c873930d37